core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -306,18 +428,19 @@ class LongAccumulator extends AccumulatorV2[jl.Long, jl.Long] {
 
   /**
    * Adds v to the accumulator, i.e. increment sum by v and count by 1.
-   * @since 2.0.0
+   * Added for simplicity with adding non java Longs.
+   * @since 2.1.0
    */
-  override def add(v: jl.Long): Unit = {
-    _sum += v
-    _count += 1
+  def add(v: Long): Unit = {
```

Comment:
So we could do that, but it seems like maybe having the specialized is better than copying that code? I don't have super strong feelings about this but just something to consider.